### DISCUSSION TITLE
Which apk version should I download?

### DISCUSSION THREAD

**Discussion by Eta-Beta** [Alex313031/Thorium-Android#31] [General]
Good morning, I'm not very expert and I'm undecided on which apk I should download for my Xiaomi Poco M4 Pro. Thank you for your help.

**Comment by gz83** [comment]
Please consider using the arm64 version

**Comment by Eta-Beta** [comment]
Good morning, the version of Thorium is installed regularly, I only have another small problem, my smartphone account won't accept it.  When I enter it it says "account already present on device".  In my Xiaomi I have deactivated Chrome (because uninstalling it is practically impossible without root) and I use Brave, making a comparison between the two in these 2 days of use I can say that Thorium is much faster as with the PC, but on the smartphone it does not block all the Brave hype....maybe I'll still have to setup something else.

**Comment by gz83** [comment]
This is because Google restricts Google account login on the third-party compiled Chromium browser on the Android platform

**Comment by Eta-Beta** [comment]
Good morning, unfortunately I will have to uninstall Thorium because in addition to the problem that you cannot log in with your Google account it blocks very few advertisements, unlike Brave which however is slower.  Thank you so much for your support, I hope the project develops even better
